### PR TITLE
Issue 36 main branch should always have a working version

### PR DESCRIPTION
**Pull request by rreinecke**
I've updated the readme as @tomglee suggested. It is confusing that the version visible through the main branch is not the same as the version we current distribute. I have updated both streamlit links to be ow connected to the "new" streamlit account.
I've also merge changes from the development branch into main.

From now on I would leave main be and only use dev for the next versions until we have a working product. e.g. a app running on streamlit.io that stores its data on hydroshare.

### CODE REVIEW

**Comment by rreinecke** [comment]
@kbefus if you need help on how to do the review thing. Let me know or assign someone else :)

**Comment by kbefus** [comment]
Yeah, I'm not sure what is meant by review - it'd be helpful to have an example session with you.

**Comment by rreinecke** [comment]
> Yeah, I'm not sure what is meant by review - it'd be helpful to have an example session with you.

Sure! would be happy to do that. If you like you could also have a look at this video here: https://youtu.be/lSnbOtw4izI?t=44

How else from the group would be able to potentially step in as reviewer? Juan certainly. What about Xander?

File: README.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
...
 Groundwater Model Portal — sharing regional to global groundwater model data
 
-Main release (main branch) Streamlit app running live [here](https://share.streamlit.io/juancastilla/gromopo/main/streamlit/GroMoPo_app.py)
+## Check out the current app version on Streamlit.io
+Main release (main branch) Streamlit app running live [here](https://share.streamlit.io/gromopo/gromopo/main/streamlit/GroMoPo_app.py)
+
+Bleeding edge (development branch) version Streamlit app running live [here](https://share.streamlit.io/gromopo/gromopo/develop/streamlit/GroMoPo_app.py)
+
+## Contribute to the code
+If you would like to contribute to this project check out our [guidelines](CONTRIBUTING.md)
```
Comment (kbefus) [inline README.md:10]: Comma after "project": project, check out

File: streamlit/GroMoPo_app.py [hunk 2]
```diff
@@ -22,4 +24,11 @@
                 " questions, resources and groundwater models to the source code")
 st.sidebar.title("About")
 st.sidebar.info("This app is maintained and argued on by the GroMoPo mob")
-st.sidebar.image('pages/img/GroMoPo_logo_V1.png', caption=None, width=None, use_column_width=None, clamp=False, channels='RGB', output_format='auto')
+
+if platform.system() == 'Windows':
```
Comment (kbefus) [inline streamlit/GroMoPo_app.py:28]: This is where more operating systems could included in a list for local OS's. A better solution would be to somehow test if running on streamlit or locally.

if platform.system().lower() in ['windows','otherOS']:

File: streamlit/pages/model_finder.py [hunk 3]
```diff
@@ -7,66 +7,95 @@
...
+    for img in rasters:
+    
+        rgroup = folium.FeatureGroup(name=rasters[img]['name'],overlay=True)
+        rgroup.add_child(folium.raster_layers.ImageOverlay(rasters[img]['data'], opacity=0.6,
+                                                           bounds=[[-90, -180], [90, 180]],
+                                                           mercator_project=True))
+        rgroups.append(rgroup)
+    
+    marker_cluster = plugins.MarkerCluster(control=False)
+    mlayers = []
```
Comment (kbefus) [inline streamlit/pages/model_finder.py:29]:
```suggestion
```

File: streamlit/pages/model_finder.py [hunk 4]
```diff
@@ -7,66 +7,95 @@
...
+    mlayer=folium.GeoJson(shp_df, name='Groundwater models', popup=popup,
+                          style_function = lambda feature: {
+                            'fillColor': 'grey',
+                            'color':feature['properties']['color'],
+                            'weight': 1,
+                            'fillOpacity': 0.7,
     })
-    return rgroup, marker_cluster, mlayer    
+    
+    mlayers.append(mlayer)
```
Comment (kbefus) [inline streamlit/pages/model_finder.py:42]:
```suggestion
```

File: streamlit/pages/model_finder.py [hunk 5]
```diff
@@ -79,11 +108,26 @@ def read_img(fname, skip_rows=60):
...
+
+# Can merge, but better to keep separate
+# all_gdf = gpd.GeoDataFrame(gpd.pd.concat([all_gdf,ptsq_df],ignore_index=True))
+# all_gdf.set_crs(epsg, allow_override=True, inplace=True)
+
+# Separate datasets for models with and without domain shps
+
+# shp_dict = {'wdomains':all_gdf,
+#             'woutdomains':ptsq_df,
+#             }
```
Comment (kbefus) [inline streamlit/pages/model_finder.py:130]: I'm not sure if it's worth keeping all of these commented/failed attempts.

File: streamlit/pages/model_finder.py [hunk 6]
```diff
@@ -92,19 +136,47 @@ def app():
...
+    # modelgroup2.add_child(mlayers[1])    
+    
+    # for ishp in mlayers:
+    #     ishp.add_to(map)
+    
+    # modelgroup.add_to(map)
+    # modelgroup1.add_to(m)
+    # modelgroup2.add_to(m)
+    
+    # folium.LayerControl().add_to(m)
```
Comment (kbefus) [inline streamlit/pages/model_finder.py:175]: More failed attempts that can be deleted.

**Review by kbefus — DISMISSED** [review]
Looks good. No major changes needed. Some suggestions.

File: streamlit/GroMoPo_app.py (hunk 2 above)
Comment (rreinecke) [reply streamlit/GroMoPo_app.py:28]: I added a comment to track this better.

**Review by kbefus — APPROVED** [review]
Looks good!

**Pull request merged**